.github/workflows/build-crates-individually.yml

```diff
@@ -0,0 +1,103 @@
+name: Build crates individually
+
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - main
+    paths:
+      # production code and test code
+      - '**/*.rs'
+      # dependencies
+      - '**/Cargo.toml'
+      - '**/Cargo.lock'
+      # workflow definitions
+      - '.github/workflows/build-crates-individually.yml'
```

Comment:
Changes to these files can also affect the results, because they change warnings, and some code is compiled differently in individual crates:
- https://github.com/ZcashFoundation/zebra/blob/main/.cargo/config.toml
- https://github.com/ZcashFoundation/zebra/blob/main/clippy.toml

I think we might want to open a ticket to add these files to all our CI.

Reply:
This is now ticket #4652.